Watching for file changes with StatReloader
Not Found: /
[21/Mar/2020 09:06:01] "GET / HTTP/1.1" 404 2037
Not Found: /
[21/Mar/2020 09:06:19] "GET / HTTP/1.1" 404 2037
Not Found: /
[21/Mar/2020 09:06:27] "GET / HTTP/1.1" 404 2037
[21/Mar/2020 09:06:35] "GET /products HTTP/1.1" 301 0
[21/Mar/2020 09:06:35] "GET /products/ HTTP/1.1" 200 10
[21/Mar/2020 09:06:39] "GET /products/new HTTP/1.1" 200 11
[21/Mar/2020 09:06:43] "GET /admin HTTP/1.1" 301 0
[21/Mar/2020 09:06:43] "GET /admin/ HTTP/1.1" 200 4070
[21/Mar/2020 09:06:44] "GET /static/admin/css/base.css HTTP/1.1" 304 0
[21/Mar/2020 09:06:44] "GET /static/admin/css/responsive.css HTTP/1.1" 304 0
[21/Mar/2020 09:06:44] "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
[21/Mar/2020 09:06:44] "GET /static/admin/css/fonts.css HTTP/1.1" 304 0
[21/Mar/2020 09:06:44] "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
[21/Mar/2020 09:06:44] "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
[21/Mar/2020 09:06:44] "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
[21/Mar/2020 09:06:44] "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
[21/Mar/2020 09:06:44] "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
[21/Mar/2020 09:06:46] "GET /admin/products/product/ HTTP/1.1" 200 4727
[21/Mar/2020 09:06:46] "GET /static/admin/css/changelists.css HTTP/1.1" 304 0
[21/Mar/2020 09:06:46] "GET /static/admin/js/jquery.init.js HTTP/1.1" 304 0
[21/Mar/2020 09:06:46] "GET /admin/jsi18n/ HTTP/1.1" 200 3223
[21/Mar/2020 09:06:46] "GET /static/admin/js/core.js HTTP/1.1" 304 0
[21/Mar/2020 09:06:46] "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 304 0
[21/Mar/2020 09:06:46] "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 304 0
[21/Mar/2020 09:06:46] "GET /static/admin/js/actions.js HTTP/1.1" 304 0
[21/Mar/2020 09:06:46] "GET /static/admin/js/urlify.js HTTP/1.1" 304 0
[21/Mar/2020 09:06:46] "GET /static/admin/js/prepopulate.js HTTP/1.1" 304 0
[21/Mar/2020 09:06:46] "GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 304 0
[21/Mar/2020 09:06:47] "GET /static/admin/img/tooltag-add.svg HTTP/1.1" 304 0
[21/Mar/2020 09:08:02] "GET /admin/products/product/add/ HTTP/1.1" 200 5409
[21/Mar/2020 09:08:03] "GET /admin/jsi18n/ HTTP/1.1" 200 3223
[21/Mar/2020 09:08:52] "POST /admin/products/product/add/ HTTP/1.1" 302 0
[21/Mar/2020 09:08:52] "GET /admin/products/product/ HTTP/1.1" 200 5215
[21/Mar/2020 09:08:53] "GET /admin/jsi18n/ HTTP/1.1" 200 3223
[21/Mar/2020 09:08:53] "GET /static/admin/img/icon-yes.svg HTTP/1.1" 304 0
[21/Mar/2020 09:09:24] "GET /admin/products/product/add/ HTTP/1.1" 200 5409
[21/Mar/2020 09:09:24] "GET /admin/jsi18n/ HTTP/1.1" 200 3223
[21/Mar/2020 09:10:02] "POST /admin/products/product/add/ HTTP/1.1" 302 0
[21/Mar/2020 09:10:02] "GET /admin/products/product/ HTTP/1.1" 200 5497
[21/Mar/2020 09:10:03] "GET /admin/jsi18n/ HTTP/1.1" 200 3223
/usr/local/src/PyShop/products/admin.py changed, reloading.
Performing system checks...

System check identified no issues (0 silenced).
March 21, 2020 - 09:05:46
Django version 3.0.4, using settings 'pyshop.settings'
Starting development server at http://0.0.0.0:7777/
Quit the server with CONTROL-C.
Watching for file changes with StatReloader
[21/Mar/2020 09:13:07] "GET /admin/products/ HTTP/1.1" 200 2899
[21/Mar/2020 09:13:10] "GET /admin/ HTTP/1.1" 200 5103
[21/Mar/2020 09:13:11] "GET /admin/ HTTP/1.1" 200 5103
[21/Mar/2020 09:13:36] "GET /admin/products/offer/ HTTP/1.1" 200 3170
[21/Mar/2020 09:13:36] "GET /admin/jsi18n/ HTTP/1.1" 200 3223
[21/Mar/2020 09:14:27] "GET /admin/products/offer/add/ HTTP/1.1" 200 4951
[21/Mar/2020 09:14:27] "GET /static/admin/css/forms.css HTTP/1.1" 304 0
[21/Mar/2020 09:14:27] "GET /admin/jsi18n/ HTTP/1.1" 200 3223
[21/Mar/2020 09:14:27] "GET /static/admin/js/change_form.js HTTP/1.1" 304 0
[21/Mar/2020 09:14:27] "GET /static/admin/js/prepopulate_init.js HTTP/1.1" 304 0
[21/Mar/2020 09:14:28] "GET /static/admin/css/widgets.css HTTP/1.1" 304 0
[21/Mar/2020 09:14:47] "POST /admin/products/offer/add/ HTTP/1.1" 302 0
[21/Mar/2020 09:14:49] "GET /admin/products/offer/ HTTP/1.1" 200 4736
[21/Mar/2020 09:14:51] "GET /admin/jsi18n/ HTTP/1.1" 200 3223
/usr/local/src/PyShop/products/views.py changed, reloading.
Performing system checks...

System check identified no issues (0 silenced).
March 21, 2020 - 09:12:59
Django version 3.0.4, using settings 'pyshop.settings'
Starting development server at http://0.0.0.0:7777/
Quit the server with CONTROL-C.
Watching for file changes with StatReloader
/usr/local/src/PyShop/products/views.py changed, reloading.
Performing system checks...

System check identified no issues (0 silenced).
March 21, 2020 - 09:20:49
Django version 3.0.4, using settings 'pyshop.settings'
Starting development server at http://0.0.0.0:7777/
Quit the server with CONTROL-C.
Watching for file changes with StatReloader
[21/Mar/2020 09:25:39] "GET /admin/products/offer/ HTTP/1.1" 200 4542
[21/Mar/2020 09:25:39] "GET /admin/jsi18n/ HTTP/1.1" 200 3223
Not Found: /
[21/Mar/2020 09:25:43] "GET / HTTP/1.1" 404 2037
[21/Mar/2020 09:25:52] "GET /products/ HTTP/1.1" 200 80
/usr/local/src/PyShop/products/views.py changed, reloading.
Performing system checks...

System check identified no issues (0 silenced).
March 21, 2020 - 09:25:37
Django version 3.0.4, using settings 'pyshop.settings'
Starting development server at http://0.0.0.0:7777/
Quit the server with CONTROL-C.
Watching for file changes with StatReloader
[21/Mar/2020 09:32:29] "GET /products/ HTTP/1.1" 200 117
[21/Mar/2020 09:45:19] "GET /products/ HTTP/1.1" 200 117
[21/Mar/2020 09:45:21] "GET /products/ HTTP/1.1" 200 117
[21/Mar/2020 09:45:41] "GET /products/ HTTP/1.1" 200 117
[21/Mar/2020 10:07:08] "GET /products/ HTTP/1.1" 200 117
[21/Mar/2020 10:07:10] "GET /products/ HTTP/1.1" 200 117
[21/Mar/2020 10:07:27] "GET /products/ HTTP/1.1" 200 2506
Not Found: /products/...
[21/Mar/2020 10:07:27] "GET /products/... HTTP/1.1" 404 2297
[21/Mar/2020 10:11:26] "GET /products/ HTTP/1.1" 200 2499
[21/Mar/2020 10:16:11] "GET /products/ HTTP/1.1" 200 2617
Internal Server Error: /products/
Traceback (most recent call last):
  File "/root/anaconda3/lib/python3.7/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/root/anaconda3/lib/python3.7/site-packages/django/template/base.py", line 171, in render
    return self._render(context)
  File "/root/anaconda3/lib/python3.7/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
  File "/root/anaconda3/lib/python3.7/site-packages/django/template/base.py", line 936, in render
    bit = node.render_annotated(context)
  File "/root/anaconda3/lib/python3.7/site-packages/django/template/base.py", line 903, in render_annotated
    return self.render(context)
  File "/root/anaconda3/lib/python3.7/site-packages/django/template/loader_tags.py", line 127, in render
    compiled_parent = self.get_parent(context)
  File "/root/anaconda3/lib/python3.7/site-packages/django/template/loader_tags.py", line 124, in get_parent
    return self.find_template(parent, context)
  File "/root/anaconda3/lib/python3.7/site-packages/django/template/loader_tags.py", line 104, in find_template
    template_name, skip=history,
  File "/root/anaconda3/lib/python3.7/site-packages/django/template/engine.py", line 129, in find_template
    raise TemplateDoesNotExist(name, tried=tried)
django.template.exceptions.TemplateDoesNotExist: base.html

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/root/anaconda3/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/root/anaconda3/lib/python3.7/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/anaconda3/lib/python3.7/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/src/PyShop/products/views.py", line 9, in index
    {'products': products })
  File "/root/anaconda3/lib/python3.7/site-packages/django/shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/root/anaconda3/lib/python3.7/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "/root/anaconda3/lib/python3.7/site-packages/django/template/backends/django.py", line 63, in render
    reraise(exc, self.backend)
  File "/root/anaconda3/lib/python3.7/site-packages/django/template/backends/django.py", line 84, in reraise
    raise new from exc
django.template.exceptions.TemplateDoesNotExist: base.html
[21/Mar/2020 10:58:31] "GET /products/ HTTP/1.1" 500 127838
Internal Server Error: /products/
Traceback (most recent call last):
  File "/root/anaconda3/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/root/anaconda3/lib/python3.7/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/anaconda3/lib/python3.7/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/src/PyShop/products/views.py", line 9, in index
    {'products': products })
  File "/root/anaconda3/lib/python3.7/site-packages/django/shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/root/anaconda3/lib/python3.7/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/root/anaconda3/lib/python3.7/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/root/anaconda3/lib/python3.7/site-packages/django/template/backends/django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/root/anaconda3/lib/python3.7/site-packages/django/template/engine.py", line 143, in get_template
    template, origin = self.find_template(template_name)
  File "/root/anaconda3/lib/python3.7/site-packages/django/template/engine.py", line 125, in find_template
    template = loader.get_template(name, skip=skip)
  File "/root/anaconda3/lib/python3.7/site-packages/django/template/loaders/base.py", line 30, in get_template
    contents, origin, origin.template_name, self.engine,
  File "/root/anaconda3/lib/python3.7/site-packages/django/template/base.py", line 156, in __init__
    self.nodelist = self.compile_nodelist()
  File "/root/anaconda3/lib/python3.7/site-packages/django/template/base.py", line 194, in compile_nodelist
    return parser.parse()
  File "/root/anaconda3/lib/python3.7/site-packages/django/template/base.py", line 477, in parse
    raise self.error(token, e)
  File "/root/anaconda3/lib/python3.7/site-packages/django/template/base.py", line 475, in parse
    compiled_result = compile_func(self, token)
  File "/root/anaconda3/lib/python3.7/site-packages/django/template/loader_tags.py", line 264, in do_extends
    bits[1] = construct_relative_path(parser.origin.template_name, bits[1])
  File "/root/anaconda3/lib/python3.7/site-packages/django/template/loader_tags.py", line 239, in construct_relative_path
    "template '%s' is in." % (relative_name, current_template_name)
django.template.exceptions.TemplateSyntaxError: The relative path ''../../templates/base.html'' points outside the file hierarchy that template 'index.html' is in.
[21/Mar/2020 10:59:00] "GET /products/ HTTP/1.1" 500 131360
/usr/local/src/PyShop/pyshop/settings.py changed, reloading.
Performing system checks...

System check identified no issues (0 silenced).
March 21, 2020 - 09:28:40
Django version 3.0.4, using settings 'pyshop.settings'
Starting development server at http://0.0.0.0:7777/
Quit the server with CONTROL-C.
Watching for file changes with StatReloader
[21/Mar/2020 11:03:22] "GET /products/ HTTP/1.1" 200 2617
[21/Mar/2020 11:06:04] "GET /products/ HTTP/1.1" 200 2635
